__tests__/pet.tests.js

```diff
@@ -0,0 +1,170 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+    it('returns a new object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+    });
+    it('sets intial age to 0, hunger of 0, fitness of 10', () => {
```

Comment:
Should have a clear line between each test block to make it easier to read, would be good to fix this throughout this file